modules/oe_webtools_analytics/modules/oe_webtools_analytics_rules/config/schema/webtools_analytics_rule.schema.yml

```diff
@@ -8,8 +8,11 @@ oe_webtools_analytics_rules.webtools_analytics_rule.*:
     section:
       type: label
       label: 'Section'
+    multilingual:
```

Comment:
More than multilingual, this should be `default_language` or `default_language_only` or similar.
From my understanding of the specifications, when this option is enabled the rule should be matched against the alias of the current page in the default site language. While it reflects well on multilingual, it's not really representative of the functionality.

Reply:
Let's go for the following native-approved version:

```yml
match_on_site_default_language:
  type: boolean
  label: 'Match on path alias for site default language'
```

Reply:
Done